### DISCUSSION TITLE
how to fix 'ErrorException Undefined variable: dumpFilters' at production server

### DISCUSSION THREAD

**Discussion by mahdijafaridev** [rappasoft/laravel-livewire-tables#579] [Q&A]
I git this error at the production server:
`ErrorException
Undefined variable: dumpFilters (View: /home/server/public_html/project/resources/views/vendor/livewire-tables/includes/debug.blade.php)`

It works fine on my PC but doesn't work on the server. 
how to fix this error?
thanks

**Comment by khedira00** [comment]
php artisan clear-compiled &&
php artisan cache:clear &&
php artisan route:clear &&
php artisan view:clear &&
php artisan config:clear &&
composer dumpautoload -o 
Try this commands to run

**Reply by divoto** [reply]
It doesn't fix anything.

**Comment by dhruva81** [comment]
Delete your previously published  livewire-tables directory   from `resources/views/vendor/livewire-tables`
